webv2/primary.go

```diff
@@ -0,0 +1,319 @@
+package webv2
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+	"github.com/cloudspannerecosystem/harbourbridge/webv2/session"
+)
+
+// PrimaryKeyRequest represents  Primary keys API Payload
+type PrimaryKeyRequest struct {
+	TableId      int      `json:"TableId"`
+	Columns      []Column `json:"Columns"`
+	PrimaryKeyId int      `json:"PrimaryKeyId"`
+}
+
+/*PrimaryKeyResponse represents  Primary keys API response
+Synth is true is for table Primary Key Id is not present and it is generated
+*/
+type PrimaryKeyResponse struct {
+	TableId      int      `json:"TableId"`
+	Columns      []Column `json:"Columns"`
+	PrimaryKeyId int      `json:"PrimaryKeyId"`
+	Synth        bool     `json:"Synth"`
+}
+
+type Column struct {
+	ColumnId int    `json:"ColumnId"`
+	ColName  string `json:"ColName"`
+	Desc     bool   `json:"Desc"`
+	Order    int    `json:"Order"`
+}
+
+//primaryKey updates Primary keys in Spanner Table.
+func primaryKey(w http.ResponseWriter, r *http.Request) {
+
+	reqBody, err := ioutil.ReadAll(r.Body)
```

Comment:
maybe a larger requirement, but we should add logs when we receive a request.

Reply:
log.Println("request started", "traceid", id.String(), "method", r.Method, "path", r.URL.Path, "remoteaddr", r.RemoteAddr)